tests/image_compare.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env python3
+
+from PIL import Image
+import numpy as np
+import sys
+
+
+def Compare3x3(img1, img2):
```

Comment:
It wouldn't hurt with a short explanation what this actually does, as this code is pretty hard to read.

Reply:
Right.  Done.

Reply:
Awesome!